gremlin-go/driver/graphTraversal.go

```diff
@@ -669,6 +670,30 @@ func (g *GraphTraversal) Write(args ...interface{}) *GraphTraversal {
 	return g
 }
 
+func int32Args(args ...interface{}) []interface{} {
+	for i, arg := range args {
+		switch val := arg.(type) {
+		case uint8:
+			args[i] = int32(val)
+		case uint32:
+			args[i] = int32(val)
+		case uint64:
+			args[i] = int32(val)
+		case int8:
+			args[i] = int32(val)
+		case int16:
+			args[i] = int32(val)
+		case int64:
+			args[i] = int32(val)
+		case int:
+			args[i] = int32(val)
```

Comment:
`uint16` and `uint` are missing. Is it intended?

Other than that, I remember running into this problem when I started using gremlin-go and neither the solution nor the error were obvious. I ended up greping the repo to find a solution. So, in my opinion, this is a good change for people new to gremlin-go. However, I would update the `Times` documentation to mention that the passed value will be casted to `int32`.

Reply:
`uint16` is serialized as an `int32` so that was intentional. Not sure about `uint`.